README.md

```diff
@@ -157,7 +159,66 @@ end.
    make test
    ```
 
-### 1.3.1 For personal deployment (AWS credentials available)
+### 1.3.1 AWS credentials
+
+```
+TBD
+```
+
+### 1.3.2 Google credentials
+
+1. Ask to be invited to the google cloud `human-cell-atlas-travis-test` project
+
+2. Log into `console.cloud.google.com`, select `human-cell-atlas-travis-test` project
+
+3. Navigate to the `IAM & admin` page, locate your account in list, take note of the email address found in the `Member` column (eg. alice@example.com)
+
+   ```
+   Google Cloud Platform -> Navigation menu -> IAM & admin
+   ```
+
+4. Create a service account for project `human-cell-atlas-travis-test`
+
+    ```
+    IAM & admin -> Service Accounts -> [Create Service Account]
+
+    Step 1:
+    Service Account Name: (use username part of email address noted in step 3 eg. alice)
+    Service Account ID: (use auto-generated value eg. alice-42@human-cell-atlas-travis-test.iam.gserviceaccount.com)
+    [Create]
+    
+    Step 2:
+    Role: Project -> Owner
+    [Continue]
+    
+    Step 3:
+    [Create Key] -> JSON -> [Create] -> (Download & save json file) -> [Done]
+    ```
+
+5. Move the saved json file to a safe location on your computer
+
+    ```
+    $ mkdir /Users/alice/.gcp
+    $ mv /Users/alice/Downloads/human-cell-atlas-travis-test-180b575fe2e6.json /Users/alice/.gcp/
+    ```
+    
+6. Edit your deployment's `environment.local` file, uncomment and modify the GOOGLE variables
+
+    ```
+    $ vim /Users/alice/azul/deployments/alice.local/environment.local
+    
+    export GOOGLE_APPLICATION_CREDENTIALS="/Users/alice/.gcp/human-cell-atlas-travis-test-180b575fe2e6.json"
+    export GOOGLE_PROJECT="human-cell-atlas-travis-test"
+    ```
+
+7. Create a symlink to your deployment `environment.local` file in other deployments as needed
```

Comment:
Try to not force people to use symlinks. They may already have symlinks, they may already have copies. Your instructions should work in both cases.